### PR TITLE
feat(i18n): add French translations

### PR DESCRIPTION
**Pull request by lemoustachiste**
### Summary

Provide a general description of the code changes in your pull request... were there any bugs you had fixed? If so, mention them. If these bugs have open GitHub issues, be sure to tag them here as well, to keep the conversation linked together.

### Other Information

If there's anything else that's important and relevant to your pull request, mention that information here. This could include benchmarks, or other information.

If you are updating any of the CHANGELOG files or are asked to update the CHANGELOG files by reviewers, please add the CHANGELOG entry at the top of the file.

Thanks for contributing!

### CODE REVIEW

File: src/data/i18n.json [hunk 1]
```diff
@@ -72,6 +72,79 @@
...
+    },
+    "errors": {
+      "certificateNotValid": "Certificat invalide",
+      "getChain": "Valeur de chaine non reconnue",
+      "getTransactionId": "Impossible de vérifier ce certificat sans un identifiant de transaction valide",
+      "getIssuerProfile": "Profil de l'émetteur indisponible",
+      "getRevokedAssertions": "Impossible d'obtenir les raisons de revocation",
+      "lookForTxInvalidChain": "Chaine invalide : non liée à un BlockchainExplorers connu",
+      "lookForTxInvalidAppConfig": "Configuration de l'application invalide : vérifiez la valeur de configuration de CONFIG.MinimumBlockchainExplorers",
+      "lookForTxCouldNotConfirm": "Impossible de confirmer transaction. Aucune API blockchain n'a répondu. Potentiellement du à une limite de débit réseau.",
```
Comment (raiseandfall) [inline src/data/i18n.json]: `Impossible de confirmer la transaction. Aucune API blockchain n'a répondu. Potentiellement dû à une limite de débit réseau.` ?

File: src/data/i18n.json [hunk 2]
```diff
@@ -72,6 +72,79 @@
...
+      "lookForTxDifferentAddresses": "Adresses d'émission reçues de l'API blockchain ne concordent pas",
```
Comment (raiseandfall) [inline src/data/i18n.json]: `Les adresses d'émission reçues de l'API blockchain ne concordent pas` ?

File: src/data/i18n.json [hunk 3]
```diff
@@ -72,6 +72,79 @@
...
+      "lookForTxDifferentRemoteHashes": "Hashs distants reçus de l'API blockchain ne concordent pas",
```
Comment (raiseandfall) [inline src/data/i18n.json]: `Les hashs distants reçus de l'API blockchain ne concordent pas` ?

File: src/data/i18n.json (hunk 1 above)
Comment (lemoustachiste) [reply src/data/i18n.json]: 👍

File: src/data/i18n.json (hunk 2 above)
Comment (lemoustachiste) [reply src/data/i18n.json]: 👍

File: src/data/i18n.json (hunk 3 above)
Comment (lemoustachiste) [reply src/data/i18n.json]: 👍

**Pull request merged**